### PR TITLE
Most-133: deploy scripts for l2

### PR DESCRIPTION
**Pull request by kostekIV**

### CODE REVIEW

File: eth/scripts/1_l2_setup_bridge_contracts.js [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+async function most_add_bazero_pair(most, bazeroAddress) {
+  const bazeroAddressBytes = ethers.zeroPadValue(
+    ethers.getBytes(bazeroAddress),
+    32,
+  );
+  /// TODO: switch this for actual address of wrapped azero on L1
+  const destBazeroAddressBytes = bazeroAddressBytes;
+
+  console.log("Adding `Bazero` pair to the most");
+  await most.addPair(bazeroAddressBytes, destBazeroAddressBytes, false);
```
Comment (Marcin-Radecki) [inline eth/scripts/1_l2_setup_bridge_contracts.js:96]: Can we please log `from` and `to` pair addresses?

**Review by Marcin-Radecki — APPROVED** [review]

File: eth/scripts/0_l2_deploy_bridge_contracts.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+/// Deploys `StableSwapLP` and `StableSwapTwoPool`. Sets minter of the StableSwapLp tp the pool address.
+async function deployStableSwap() {
+  const LP = await ethers.getContractFactory("StableSwapLP");
+  const lp = await LP.deploy();
+  await lp.waitForDeployment();
+
+  console.log("LP deployed to:", lp.target);
+  const Pool = await ethers.getContractFactory("StableSwapTwoPool");
+  const pool = await Pool.deploy();
+  await pool.waitForDeployment();
```
[the remark this answers is not in the document]
Comment (kostekIV) [reply eth/scripts/0_l2_deploy_bridge_contracts.js:16]: StableSwap and Most can be vierified. You need to do it by hand tho

File: eth/scripts/1_l2_setup_bridge_contracts.js [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+}
+
+/// 1. Mint Bazero.
+/// 2. Sets minter for LpToken to Pool.
+/// 3. Sets minter for bazero to Most.
+/// 4. Initialize pool, providing it with some initial liquidity.
+/// 5. Add Bazero pair to the most contract.
+/// 6. Unpause the most.
+async function main() {
```
[the remark this answers is not in the document]
Comment (kostekIV) [reply eth/scripts/1_l2_setup_bridge_contracts.js:105]: I will just add a check at the start of the script

File: eth/scripts/1_l2_setup_bridge_contracts.js (hunk 1 above)
Comment (kostekIV) [reply eth/scripts/1_l2_setup_bridge_contracts.js:113]: done

File: eth/scripts/1_l2_setup_bridge_contracts.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (kostekIV) [reply eth/scripts/1_l2_setup_bridge_contracts.js:122]: done

File: eth/scripts/1_l2_setup_bridge_contracts.js [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+  return BigInt(amount) * BigInt(10 ** decimals);
+}
+
+async function mintInitialBazero(bazero, to, config) {
+  const mintAmount = tokenToRaw(
+    config.bazero_token_mint,
+    config.bazero_decimals,
+  );
+
+  console.log(`Minting ${mintAmount} of Bazero to account ${to}`);
```
[the remark this answers is not in the document]
Comment (kostekIV) [reply eth/scripts/1_l2_setup_bridge_contracts.js:57]: It would be nice but I kinda dont want to do it :P

**Review by Marcin-Radecki — APPROVED** [review]

**Pull request merged**